pkg/internal/testhelpers/jaeger_container.go

```diff
@@ -0,0 +1,79 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license
+
+package testhelpers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/go-connections/nat"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	jaegerImage        = "jaegertracing/all-in-one:latest"
```

Comment:
This image is built from `master` branch. As such it can be unstable and not a good candidate for use in tests due to lack of reproducibility.
Can we switch it to a tagged image and allow overriding during test execution? This way we can set it in `Makefile` or github action and have good visibility into what image is used.

Reply:
This is done knowingly. We want the most recent image of jaeger. Its better to fail for latest image than to pass on deprecated behaviour.